### PR TITLE
Update dependencies and move build outputs out of vcs

### PR DESCRIPTION
**Pull request by reallyimeric**

### CODE REVIEW

**Comment by baizhenyu** [comment]
Please update `package.json` file and bump all dependencies to latest version

**Comment by trails-game** [comment]
Please bump all dependcies in `package.json` to latest version

File: package.json [hunk 1]
```diff
@@ -30,23 +30,23 @@
...
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.0.0",
     "webpack": "^5.73.0",
-    "webpack-cli": "^4.10.0"
+    "webpack-cli": "^5.1.1"
   },
   "dependencies": {
     "3d-force-graph": "^1.70.10",
```
Comment (trails-game) [inline package.json:47]: This is the core compoent that needs a bump

Comment (reallyimeric) [reply package.json:47]: The lockfile has been changed for that already. A `npm install` or `npm ci` will produce an install with version `1.71.4` of `3d-force-graph`.

**Comment by trails-game** [comment]
Please revert commit bb3f5967d8045c19271bb949064896bd31376389

**Comment by reallyimeric** [comment]
> Please revert commit [bb3f596](https://github.com/trails-game/relation-graph-3d-force/commit/bb3f5967d8045c19271bb949064896bd31376389)

Including dist dir into code may not be a good idea. How about we create the directory at runtime?

**Comment by trails-game** [comment]
> > Please revert commit [bb3f596](https://github.com/trails-game/relation-graph-3d-force/commit/bb3f5967d8045c19271bb949064896bd31376389)
> 
> Including dist dir into code may not be a good idea. How about we create the directory at runtime?

The demo page relies on dist artifact. We want to show the demo locally without a working building environment.

**Review by trails-game — APPROVED** [review]

**Pull request merged**